integration-tests/api/__tests__/admin/currency.js

```diff
@@ -54,39 +54,6 @@ describe("/admin/currencies", () => {
       expect(response.data).toMatchSnapshot()
     })
   })
-
-  describe("POST /admin/currencies/:code", function () {
```

Comment:
comment: As of 0.14.0, the `class-validator` forbids unknown values by default. This test is therefore redundant, as the `includes_tax` property will not be allowed payload when the FF is disabled.

Reply:
https://github.com/typestack/class-validator/blob/develop/CHANGELOG.md#breaking-changes